### PR TITLE
Document command-line flag for delaying enrollment

### LINKED ISSUE #918: [Request] Document delayed enrollment for gold images and VDIs
## Description

The new support for `--delay-enroll`. Which will cause installation of the Elastic Agent to not enroll, but to enroll on first start of the machine.

https://github.com/elastic/beats/pull/27118

## Collaboration

* [ ] The docs team will lead producing the content
* [ ] The product team will provide the initial content and the docs team will edit / review
* [ ] The docs team will define with product team the structure and location, and the product team will provide the initial content
* [ ] Other (please describe)

## Suggested Target Release

7.15

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #918 

I tried to be somewhat generic here so that we are not committing to specific types of images.

### CODE REVIEW

File: docs/en/ingest-management/commands.asciidoc [hunk 1]
```diff
@@ -210,6 +212,11 @@ verification.
 `--certificate-authorities <string>`::
 Comma-separated list of root certificates used for server verification.
 
+`--delay-enroll`::
+Delays enrollment to occur on first start of the {agent} service. This setting
+is useful when there is no ability to run {agent} at installation time, for
```
Comment (blakerouse) [inline docs/en/ingest-management/commands.asciidoc:217]:
```suggestion
is useful when the install doesn't want the {agent} to start until next reboot or manual start of the service, for
```

**Review by blakerouse — APPROVED** [review]
Looks good.

File: docs/en/ingest-management/commands.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/en/ingest-management/commands.asciidoc:217]: Made a couple of minor edits to this statement to remove anthropomorphism (the install can't want things). Hope that's OK! :-)

File: docs/en/ingest-management/commands.asciidoc [hunk 2]
```diff
@@ -210,6 +212,11 @@ verification.
 `--certificate-authorities <string>`::
 Comma-separated list of root certificates used for server verification.
 
+`--delay-enroll`::
+Delays enrollment to occur on first start of the {agent} service. This setting
+is useful when you want the {agent} to start after the next reboot or manual start of the service, for
```
Comment (dedemorton) [inline docs/en/ingest-management/commands.asciidoc:217]:
```suggestion
is useful when you don't want the {agent} to start until after the next reboot or manual start of the service, for
```

File: docs/en/ingest-management/commands.asciidoc [hunk 3]
```diff
@@ -210,6 +212,11 @@ verification.
 `--certificate-authorities <string>`::
 Comma-separated list of root certificates used for server verification.
 
+`--delay-enroll`::
+Delays enrollment to occur on first start of the {agent} service. This setting
+is useful when you don't want the {agent} to start until after the next reboot or manual start of the service, for
```
Comment (dedemorton) [inline docs/en/ingest-management/commands.asciidoc:217]:
```suggestion
is useful when you don't want the {agent} to enroll until after the next reboot or manual start of the service, for
```

**Pull request merged**